### ISSUE TITLE
parsing error with some feeds - unichr() arg not in range(0x10000)

### ISSUE THREAD

**Issue by DinizBortolotto** [kurtmckee/feedparser#70]
Hi!
I'm receiving a unichr() arg not in range(0x10000) error when parsing some RSS feeds.

My OS/config:
Linux 4.4.0 SMP Mon Jan 11 22:30:29 CST 2016 x86_64, 
Slackware 14.2 (current), Python 2.7.11, and feedparser 5.2.1

Below is the output of parse:

```
diniz@darkstar:~$ python
Python 2.7.11 (default, Dec  6 2015, 14:10:30) 
[GCC 5.2.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import feedparser
>>> feedparser.parse('http://feeds.feedburner.com/podcast30min')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/site-packages/feedparser.py", line 3964, in parse
    feedparser.feed(data.decode('utf-8', 'replace'))
  File "/usr/lib64/python2.7/site-packages/feedparser.py", line 2124, in feed
    sgmllib.SGMLParser.feed(self, data)
  File "/usr/lib64/python2.7/sgmllib.py", line 104, in feed
    self.goahead(0)
  File "/usr/lib64/python2.7/sgmllib.py", line 186, in goahead
    self.handle_charref(name)
  File "/usr/lib64/python2.7/site-packages/feedparser.py", line 734, in handle_charref
    text = unichr(c).encode('utf-8')
ValueError: unichr() arg not in range(0x10000) (narrow Python build)
>>> quit()

```

**Comment by DinizBortolotto** [comment]
I looked for the handle_charref (line 734) in feedparser.py to understand what is happening and see that it is a kind of special character handling (cedilha and accents, for example, that are identified in the XML code by & # prefix) whose range in hexadecimal goes to 0x10000 (65,536 in decimal). Right?

So, after discovering that, I downloaded the RSS feed in XML (http://feeds.feedburner.com/podcast30min) and looked for that prefix and those would be greater than 0x10000 limit and thus confirmed the existence of characters that the handler (feedparser) does not know how to "translate" because there is no equivalent character.

The error refers to the code "& # x1f642" (in hex 1f642 is greater than 10,000) that represents the emoji smiley (https://s.w.org/images/core/emoji/72x72/1f642.png) that appears next to the phrase / link that in portuguese says, "Já doou no Catarse e quer as novas recompensas? É só doar a diferença agora".
I made a local test removing the emoticon and the handler / feedparser ran smoothly.
